rskj-core/src/test/java/co/rsk/peg/RegisterBtcTransactionIT.java

```diff
@@ -185,11 +252,57 @@ private BtcTransaction createPegInTransaction(Address federationAddress, Coin co
         return btcTx;
     }
 
-    private void assertLogPegInBtc() {
-        Sha256Hash peginTransactionHash = bitcoinTransaction.getHash();
-        List<DataWord> encodedTopics = getEncodedTopics(BridgeEvents.PEGIN_BTC.getEvent(), rskReceiver.toString(), peginTransactionHash.getBytes());
+    private BtcTransaction createMultiSigPegInTransaction(Address federationAddress, Coin coin) {
+        BtcTransaction btcTx = new BtcTransaction(btcNetworkParams);
+        btcTx.addInput(
+            BitcoinTestUtils.createHash(1),
+            0,
+            ScriptBuilder.createP2SHMultiSigInputScript(null, federationSupport.getActiveFederation().getRedeemScript())
+        );
+        btcTx.addOutput(new TransactionOutput(btcNetworkParams, btcTx, coin, federationAddress));
```

Comment:
```suggestion
    private BtcTransaction createMultiSigPegInTransaction(Address federationAddress, Coin coin) {
        BtcTransaction btcTx = new BtcTransaction(btcNetworkParams);
        
        Script p2shMultiSigInputScript = ScriptBuilder.createP2SHMultiSigInputScript(null, federationSupport.getActiveFederation().getRedeemScript())
        );
        btcTx.addInput(
            BitcoinTestUtils.createHash(1),
            0,
            p2shMultiSigInputScript;
        btcTx.addOutput(new TransactionOutput(btcNetworkParams, btcTx, coin, federationAddress));
```

no big deal, but i think it adds clarity